scripts/cvmfs/setup-releases.sh

```diff
@@ -50,7 +50,7 @@ elif [[ "$(cat /etc/os-release | grep -E '^ID=')" = 'ID=ubuntu' ]]; then
     if [ $? -ne 0 ]; then
       return 1
     fi
-    k4path="/cvmfs/sw.hsf.org/key4hep/releases/$rel/x86_64-ubuntu22.04-gcc11.3.0-opt"
+    k4path="/cvmfs/sw.hsf.org/key4hep/releases/$rel/x86_64-ubuntu22.04-gcc11.4.0-opt"
```

Comment:
```suggestion
    k4path="/cvmfs/sw.hsf.org/key4hep/releases/$rel/x86_64-ubuntu22.04-gcc11.?.0-opt"
```
Do wildcards work here? Or do we not need backwards compatibility?

Reply:
Yap, `?` works as well.  It's probably better also for the forward compatibility.

Reply:
I tried and `?` doesn't seem to work, maybe because `k4path` is expanded and not directly there. Since we only have 11.4.0 right now (since old nightlies are deleted), this is not a problem (and I don't think these changes happen often, probably the next change will be to move to ubuntu 24 before the compiler changes in ubuntu 22). I've just deployed the fix to cvmfs so please have a look in a few minutes

Reply:
I checked, now all test pass :)